tests/TestCase/TestSuite/IntegrationTestCaseTest.php

```diff
@@ -178,6 +178,7 @@ public function testAssertRedirect()
         $this->_response = new Response();
         $this->_response->header('Location', 'http://localhost/tasks/index');
 
+        $this->assertRedirect();
```

Comment:
I also though about it some more on the train back. It would also fit to how other methods like `allow()`, `deny()` etc work, when not passing the first argument. They default to the generic use case when not narrowed down.